_tsml_research_resources/soton/iridis/iridis_python.md

```diff
@@ -13,13 +13,13 @@ Alternatively, you can connect to one of the specific login nodes:
 - iridis5_c.soton.ac.uk
 - iridis5_d.soton.ac.uk (AMD CPU architecture)
 
-There is a Microsoft Teams group called "HPC Community" where you can ask questions.
+There is a Microsoft Teams group called ["HPC Community"](https://teams.microsoft.com/l/team/19%3a18c8baa70f8540d78455babffe11ad9c%40thread.tacv2/conversations?groupId=a0a40f99-c620-425f-8c12-a1216cf64cce&tenantId=4a5378f9-29f4-4d3e-be89-669d03ada9d8) where you can ask questions.
 
-## Windows interaction with ADA
+## Windows interaction with Iridis
 
-You need to be on a Soton network machine or have the VPN running to connect to Iridis. Connect to one of the adresses listed above.
+You need to be on a Soton network machine or have the [VPN running](https://amionvpn.soton.ac.uk/) to connect to Iridis. Connect to one of the adresses listed above.
 
-The recommended way of connecting to Iridis is using Putty as a command-line interface and WinSCP for file management.
+The recommended way of connecting to Iridis is using MobaXterm, or Putty as a command-line interface and WinSCP for file management.
```

Comment:
Never used this, we stick with a single recommendation to make helping new people easier.

Reply:
The University recommendation for windows is mobaxterm, I thought it would make sense to add it here to avoid conflicting with the iridis wiki recommendations - https://sotonac.sharepoint.com/teams/HPCCommunityWiki/SitePages/Connecting-to-Iridis5.aspx

Reply:
Can mention this in brackets after that this is the Soton recommendation, our group recommends Putty however.